sentry-resque/lib/sentry/resque.rb

```diff
@@ -43,7 +43,7 @@ def record(queue, worker, payload, &block)
               klass = if payload["class"].respond_to?(:constantize)
                 payload["class"].constantize
               else
-                Object.const_get(payload["class"])
+                Object.const_get(payload["class"].to_s)
```

Comment:
It would be faster and safer to check the type:
```suggestion
                (klass = payload["class"]).is_a?(Class) ? klass : Object.const_get(klass)
```

Reply:
Now checking type. Also removed `constantize` logic because it was a lot of gymnastics to basically use an ActiveSupport alias for `Object.const_get`.